presto-main/src/test/java/com/facebook/presto/type/TestFloatOperators.java

```diff
@@ -31,4 +31,68 @@ public void testTypeConstructor()
         assertFunction("FLOAT'Infinity'", FLOAT, Float.POSITIVE_INFINITY);
         assertFunction("FLOAT'-Infinity'", FLOAT, Float.NEGATIVE_INFINITY);
     }
+
+    @Test
+    public void testAdd()
```

Comment:
I just got feedback on my smallint/tinyint stuff that I should do more edge case testing. I don't see Float.MIN_VALUE or Float.MAX_VALUE anywhere, which means tests aren't hitting the edges.